stubs/fpdf2/fpdf/drawing.pyi

```diff
@@ -305,42 +307,44 @@ class PaintedPath:
     def __init__(self, x: int = ..., y: int = ...) -> None: ...
     def __deepcopy__(self: Self, memo) -> Self: ...
     @property
-    def style(self): ...
+    def style(self: Self) -> Self: ...
     @property
-    def transform(self): ...
+    def transform(self: Self) -> Self: ...
     @transform.setter
     def transform(self, tf) -> None: ...
     @property
-    def auto_close(self): ...
+    def auto_close(self: Self) -> Self: ...
     @auto_close.setter
     def auto_close(self, should) -> None: ...
     @property
-    def paint_rule(self): ...
+    def paint_rule(self: Self) -> Self: ...
     @paint_rule.setter
     def paint_rule(self, style) -> None: ...
     @property
-    def clipping_path(self): ...
+    def clipping_path(self: Self) -> Self: ...
     @clipping_path.setter
     def clipping_path(self, new_clipath) -> None: ...
+    @contextmanager
     def transform_group(self, transform) -> _GeneratorContextManager[Incomplete]: ...
```

Comment:
This should no longer be `_GeneratorContextManager`. See e.g. https://stackoverflow.com/q/49733699
```suggestion
    def transform_group(self, transform) -> Iterator[Incomplete]: ...
```